src/test/java/uk/gov/ons/ctp/integration/rhsvc/repository/impl/RespondentDataRepositoryIT_Test.java

```diff
@@ -0,0 +1,102 @@
+package uk.gov.ons.ctp.integration.rhsvc.repository.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Optional;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import uk.gov.ons.ctp.common.event.model.Address;
+import uk.gov.ons.ctp.common.event.model.CollectionCase;
+import uk.gov.ons.ctp.common.event.model.Contact;
+import uk.gov.ons.ctp.common.event.model.UAC;
+import uk.gov.ons.ctp.integration.rhsvc.RespondentHomeFixture;
+import uk.gov.ons.ctp.integration.rhsvc.repository.RespondentDataRepository;
+
+public class RespondentDataRepositoryIT_Test {
```

Comment:
Not sure why this test class came back to life. It doesn't work, and I deleted it earlier since I could see little value in maintaining it.  
If you do have to keep it for some reason, at least delete the unused variables.